core/services/wifi/WifiManager.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
```

Comment:
This shebang is not necessary, this will not be an executable

Reply:
Mistakenly added. Good.